### PR TITLE
[v1.1] Refine DENAS Distributed Training Settings in NLP BERT

### PR DESCRIPTION
**Pull request by tianyil1**

### CODE REVIEW

**Comment by tianyil1** [comment]
Please help review this PR. Thx. @xuechendi

**Review by Jian-Zhang — COMMENTED** [review]
LGTM

File: e2eAIOK/DeNas/nlp/utils.py [hunk 1]
```diff
@@ -175,7 +175,7 @@ def bert_create_optimizer(model, cfg):
         num_train_optimization_steps = int(
             cfg.num_train_steps / cfg.gradient_accumulation_steps) * cfg.train_epochs
         if ext_dist.my_size > 1:
-            num_train_optimization_steps = num_train_optimization_steps // ext_dist.my_size
+            num_train_optimization_steps = num_train_optimization_steps * ext_dist.my_size
```
Comment (xuechendi) [inline e2eAIOK/DeNas/nlp/utils.py:178]: @tianyil1 , using // or * seams to be a big change, can you explain here?

Comment (tianyil1) [reply e2eAIOK/DeNas/nlp/utils.py:178]: Yes, in the distributed performance test session ("ext_dist.my_size" will not be "1"), the previous setting will cause no training update on a few last training steps, which causes the performance regression on the distributed training. Thus, I fix this setting, which is tested well in my current distributed test.

Comment (xuechendi) [reply e2eAIOK/DeNas/nlp/utils.py:178]: Got it, thx

**Review by xuechendi — APPROVED** [review]
LGTM

**Pull request merged**